Packs/CommonTypes/IndicatorFields/indicatorfield-Rank.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "indicator_rank",
+    "version": -1,
+    "modified": "2022-11-07T08:33:16.160252771Z",
+    "name": "Rank",
+    "ownerOnly": false,
+    "description": "Used to display rank from different sources",
+    "cliName": "rank",
+    "type": "grid",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedToAll": true,
```

Comment:
Should it be "associatedToAll" in the current state?

Reply:
Yep, on purpose :)